src/Slider/Slider.js

```diff
@@ -838,58 +862,25 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	downLeft: function (sender, e) {
-		// checking button explicitly because it may be disabled due to value only
-		if (!this.$.buttonLeft.disabled) {
-			e.configureHoldPulse(this.holdConfig);
-			this.previous();
-		}
-	},
-
-	/**
-	* @private
-	*/
-	downRight: function (sender, e) {
-		// checking button explicitly because it may be disabled due to value only
-		if (!this.$.buttonRight.disabled) {
-			e.configureHoldPulse(this.holdConfig);
-			this.next();
-		}
-	},
-
-	/**
-	* @private
-	*/
-	holdLeft: function (sender, event) {
-		if (!this.$.buttonLeft.disabled) {
-			this.previous();
-		}
-	},
-
-	/**
-	* @private
-	*/
-	holdRight: function (sender, event) {
-		if (!this.$.buttonRight.disabled) {
-			this.next();
-		}
-	},
-
-	/**
-	* @private
-	*/
-	configureSpotlightHoldPulse: function (sender, e) {
-		if (e.keyCode === 13) {
-			e.configureHoldPulse(this.holdConfig);
+	jumpButtonTriggered: function (sender, ev) {
+		var isValidEvent = false;
+		if (!sender.disabled) {
+			if ((ev.type == 'down' || ev.type == 'holdpulse') || (ev.keyCode == 13 && ev.type == 'onSpotlightKeyDown')) {
```

Comment:
Hi @aarontam Thank you for your comment.
I understand what was your intention. I totally agree with that.
Yes, we can use `down` instead of `ondown` with your original code.

I have a class now so will update it after 9 hours. Thank :+1: